### ISSUE TITLE
Analytics: Resume bumping wpios_site_created

### ISSUE THREAD

**Issue by aerych** [wordpress-mobile/WordPress-iOS#11809]
For more context see the comment thread at p77Llu-c4u-p2
With the introduction of the new site creation flow back in March, we stopped bumping the `wpios_site_created` event in favor of `wpios_enhanced_site_creation_completed`.  

To address some confusion in the event names and other dashboards that might have been impacted, we'd like to resume bumping the `wpios_site_created` event.  We want to stop bumping `wpios_enhanced_site_creation_completed` since it will make it easier to aggregate data across both events.

### Matchin Android issue:

https://github.com/wordpress-mobile/WordPress-Android/issues/9959

**Comment by diegoreymendez** [comment]
We'll probably want to revise this change for other events as well.  I'm not sure why this change was introduced but I can see other events with the same prefix in WPiOS:

```
enhanced_site_creation_accessed
enhanced_site_creation_segments_viewed
enhanced_site_creation_segments_selected
enhanced_site_creation_verticals_viewed
enhanced_site_creation_verticals_selected
enhanced_site_creation_verticals_skipped
enhanced_site_creation_basic_information_viewed
enhanced_site_creation_basic_information_completed
enhanced_site_creation_basic_information_skipped
enhanced_site_creation_domains_accessed
enhanced_site_creation_domains_selected
enhanced_site_creation_success_loading
enhanced_site_creation_preview_viewed
enhanced_site_creation_preview_loaded
enhanced_site_creation_completed
enhanced_site_creation_error_shown
```

**Comment by aerych** [comment]
🤔 I see some value in the `enhanced_*` events as together they represent an entire feature iteration and make it easy to identify related events for funnels.  But they haven't been out in the wild very long so 🤷‍♂ .

**Comment by jkmassel** [comment]
I'm migrating this issue to `12.7` – if it was actually resolved in `12.6`, feel free to close :)

**Comment by diegoreymendez** [comment]
Moving back into 12.6 since it was merged into that version.  Thanks @jkmassel!
